src/_internals/serializer/index.spec.ts

```diff
@@ -0,0 +1,78 @@
+import Entity from '../../data/_internals/Entity';
+import SerializerFactory from './index';
+import { handle, send } from '../../channel';
+import EntityCollection from '../../data/_internals/EntityCollection';
+import Criteria from '../../data/Criteria';
+
+const { serialize, deserialize } = SerializerFactory({
+  handle: handle,
+  send: send,
+})
+
+describe('index.ts', () => {
+  it('should convert entities with circular reference', () => {
+    // @ts-ignore
```

Comment:
I think we should use `@ts-expect-error` we also do that in other projects. It also tells you that it safe to remove that directive once it no longer can ignore an error because there is no error.